test/test_depsolve.py

```diff
@@ -13,6 +13,9 @@
     "./test/data/testrepos/custom/",
 ]
 
+# osbuild-depsolve-dnf uses the GPG header to detect if keys are defined in-line or as file paths/URLs
```

Comment:
Huh, where is this behaviour implemented? Am I just missing something?

(otherwise this PR looks good)